### PR TITLE
random sonarlint cleanups

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

Mostly cosmetic. I've tested this locally and found no problems.

### CODE REVIEW

File: src/metadata/exiv2_handler.h [hunk 1]
```diff
@@ -36,12 +36,16 @@
...
+
 /// \brief This class is responsible for reading exif header metadata
 class Exiv2Handler : public MetadataHandler {
 public:
     explicit Exiv2Handler(const std::shared_ptr<Context>& context)
         : MetadataHandler(context)
     {
+        // silence annoying and pointless warnings
+        Exiv2::LogMsg::setLevel(Exiv2::LogMsg::mute);
     }
```
Comment (KarlStraussberger) [inline src/metadata/exiv2_handler.h:49]: We should make that depend on gerbera debug mode. Best to split the change into separate PR.

Comment (neheb) [reply src/metadata/exiv2_handler.h:49]: That's actually here: https://github.com/gerbera/gerbera/pull/2114 . It leaked into this commit.
